src/libostree/ostree-repo-private.h

```diff
@@ -104,6 +104,13 @@ typedef struct {
   fsblkcnt_t max_blocks;
 } OstreeRepoTxn;
 
+typedef struct {
+  GMutex mutex;
```

Comment:
Might be worth adding a comment to say that every other member of `OstreeRepoLock` should be accessed only with `mutex` held